telemetry/specs/INT.mdk

```diff
@@ -1953,18 +2074,19 @@ INT-MD over IPv4/GRE
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+  |
    |                      queue occupancy of hop1                  |  |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<-+
-   |      Inner IPv4 Header + Payload + Padding (L2/L3/ESP/….)     |
+   |                    Payload (Original Packet)                  |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
-## Example with INT-MX over IPv4/GRE
+## Example with INT-MX over IPv4/GRE (Original packet CE or IP)
 
 In this scenario host1 sends an IPv4 packet to host2.
-The ToR switch of host1 (Switch1) acts as the INT source.
-It does IPv4/GRE encapsulation and inserts INT-MX header before the
-inner (original) IPv4. Switch2 processes the INT-MX header. Finally, the ToR
+The ToR switch of host1 (Switch1) acts as the INT source. It add an Ethernet 
+(L2) header, does IPv4/GRE encapsulation and inserts INT-MX header before the
+inner (original) packet. Switch2 processes the INT-MX header. Finally, the ToR
```

Comment:
inner (original) packet that starts with an Ethernet header.

Reply:
Added